### ISSUE TITLE
inconsistent dates

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#433]
I receive inconsistent `date_insert` depending on api entry for the same resource.

Just after message POST:

```json
// GET /api/v2/messages/b6e2f697-4378-4eba-adbf-e0feb4bda86a
{
        // ...
	"date": "0001-01-01T00:00:00Z",
	"date_delete": "0001-01-01T00:00:00Z",
	"date_insert": "2017-08-09T08:37a:43.103",
        // ...
}
```

After invalidating local cache:

```json
// GET /api/v2/messages?offset=0&limit=20&discussion_id=3f4b043d-4b09-4fec-9685-b70131ad55b2

{
	"total": 1,
	"messages": [{
		// ...
	  	"date": "0001-01-01T00:00:00Z",
		"date_delete": "0001-01-01T00:00:00Z",
		"date_insert": "2017-08-09T08:37:43.103191",
		// ...
	}]
}
```

**Comment by sapiens-sapide** [comment]
Should be fixed by work in branch `improvement/backend/date-string-standardization`

**Comment by sapiens-sapide** [comment]
Fix by PR #417

**Comment by iamdey** [comment]
ok on my computer
